client/src/app/site/pages/meetings/pages/interaction/modules/interaction-container/components/video-player/video-player.component.ts

```diff
@@ -184,6 +208,45 @@ export class VideoPlayerComponent implements AfterViewInit, OnDestroy {
         this.playVjsVideo();
     }
 
+    public async updateNano(): Promise<void> {
+        let entry: any = {
+            bintu: {
+                streamid: this.videoId
+            }
+        };
+
+        // TODO: Including via rtmp streams is deprecated.
```

Comment:
Solve this now or create a cleanup issue, please.

Reply:
This needs to be included for backward compatibility. I will create an issue.